tipg/factory.py

```diff
@@ -527,6 +532,7 @@ def collections(
                 links=links,
                 numberMatched=matched_items,
                 numberReturned=items_returned,
+                lastUpdated=str(collection_catalog["last_updated"]),
```

Comment:
let's just leave this out since it's not part of the spec

Reply:
👍